javascript/javascript1/week1/age.js

```diff
@@ -0,0 +1,69 @@
+let yearOfBirth = 1995;
+let yearFuture = 2027;
+let age = yearOfBirth - yearFuture;
+console.log("You will be " + age + "years old in " + yearFuture + ".");
```

Comment:
When I run this I get `"You will be -32years old in 2027."` I don\t think my age is -32. Could you make this number positive? 
It would also be nice to have a space between the number and the word 'years'